src/Mono.Android/metadata

```diff
@@ -1699,7 +1699,15 @@
   <!-- This method was already specified on a base interface, ignore to avoid "breakage" report. -->
   <remove-node api-since="33" path="/api/package[@name='java.util']/interface[@name='Deque']/method[@name='addAll' and count(parameter)=1 and parameter[1][@type='java.util.Collection&lt;? extends E&gt;']]" />
 
-	<!--
+  <!-- Rename Namespaces -->
+  <attr api-since="33" path="/api/package[@name='android.os.vibrator']" name="managedName">Android.OS.Vibrators</attr>
```

Comment:
Does `<ns-replace/>` support `api-since`?  *Should it*?

Reply:
It does not.  However `<ns-replace />` will not throw a warning if doesn't match anything, like `metadata` will.  😁